File-Formats/PDF/Split-Documents.md

```diff
@@ -0,0 +1,276 @@
+
+Refer to the following code example to split a PDF into individual pages.
+
+{% tabs %}  
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Due to platform limitations, Essential PDF supports splitting a PDF file into individual pages only in Windows Forms, WPF, ASP.NET, and ASP.NET MVC platforms. However this can be achieved by using the following code snippet. 
+
+//Load the PDF document
+FileStream docStream = new FileStream("Input.pdf", FileMode.Open, FileAccess.Read);
+PdfLoadedDocument loadedDocument = new PdfLoadedDocument(docStream);
+for (int i=0;i<loadedDocument.PageCount;i++)
+{
+//Creates a new document
+PdfDocument document = new PdfDocument();
+//Imports the pages from the loaded document
+document.ImportPage(loadedDocument, i);
+
+//Create a memory stream 
+MemoryStream stream = new MemoryStream();
+//Save the document to stream
+document.Save(stream);
+//Close the document
+document.Close(true);
+}
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+
+//Load the PDF document
+PdfLoadedDocument loadedDocument = new PdfLoadedDocument("Input.pdf");
+//Set a output path
+const string destinationFilePattern = "Output" + "{0}.pdf";
+//Split the pages into separate documents
+loadedDocument.Split(destinationFilePattern);
+//Close the document
+loadedDocument.Close(true);
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+'Load the PDF document
+Dim loadedDocument As New PdfLoadedDocument("Input.pdf")
+'Set a output path
+Const destinationFilePattern As String = "Output" + "{0}.pdf"
+'Split the pages into separate documents
+loadedDocument.Split(destinationFilePattern)
+'Close the document
+loadedDocument.Close(True)
+
+{% endhighlight %}
+
+{% endtabs %}  
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/Pages/Splitting-PDF-file-into-individual-pages/). 
+
+## Split a range of pages into a separate PDF document
+
+The Syncfusion .NET PDF library allows splitting a certain range of pages into a separate PDF document using the [SplitByRanges](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedDocument.html#Syncfusion_Pdf_Parsing_PdfLoadedDocument_SplitByRanges_System_String_System_Int32_0__0___) method of the [PdfLoadedDocument](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedDocument.html) class. 
+
+Refer to the following code example to split a range of pages.
+
+{% tabs %}  
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+//Due to platform limitations, Essential PDF supports splitting a range of pages into separate PDF document only in Windows Forms, WPF, ASP.NET, and ASP.NET MVC platforms. However this can be achieved by using the following code snippet. 
```

Comment:
Split by Ranges supports cross-platform too. You can use it like.
```
//Load the existing PDF file.
PdfLoadedDocument loadDocument = new PdfLoadedDocument(new FileStream("input.pdf", FileMode.Open));

//Subscribe to the document split event.
loadDocument.DocumentSplitEvent += LoadDocument_DocumentSplitEvent;

void LoadDocument_DocumentSplitEvent(object sender, PdfDocumentSplitEventArgs args)
{
    //Save the resulting document.
    FileStream outputStream = new FileStream(Guid.NewGuid().ToString() + ".pdf", FileMode.CreateNew);
    args.PdfDocumentData.CopyTo(outputStream);
    outputStream.Close();
}

//Spit the document by ranges.
loadDocument.SplitByRanges(new int[,] { { 0, 5 }, { 5, 10 } }) ;

//Close the document.
loadDocument.Close(true);
```

Reply:
Changed code like this